src/docs/codelabs/index.md

```diff
@@ -3,109 +3,147 @@ title: Codelabs
 description: "Codelabs help you quickly get started programming Flutter."
 ---
 
+The Flutter codelabs provide a guided,
+hands-on coding experience. Some codelabs
+run in DartPad&mdash;no download required!
 
-#### [Basic Flutter layout concepts](/docs/codelabs/layout-basics)
-
-Use DartPad in a browser (no need to download Flutter or Dart!)
-to learn the basics of creating a Flutter layout.
 
+#### [Write your first Flutter app, part 1][]
 
-#### [Write Your First Flutter App, part 1]({{site.codelabs}}/codelabs/first-flutter-app-pt1)
+Create a simple mobile app that generates proposed names
+for a startup company. In part one, you'll use a package
+that returns pairs of words at random and inserts them into
+an infinite scrolling list. You can also find this
+[codelab on flutter.dev][].
 
-Create a simple mobile app that generates proposed names for a startup
-company. In part one, you'll use a package that returns pairs of words
-at random and inserts them into an infinite scrolling list.
 
 
-#### [Write Your First Flutter App, part 2]({{site.codelabs}}/codelabs/first-flutter-app-pt2)
+#### [Write your first Flutter app, part 2][]
 
-Create a simple mobile app that generates proposed names for a startup
-company. In part two, you'll extend the example from part 1 to allow
-the user to select favorite word pairs, and add a second "Saved Favorites"
+Create a simple mobile app that generates proposed names
+for a startup company. In part two, you'll extend the
+example from part 1 to allow the user to select favorite
+word pairs, and add a second "Saved Favorites"
 page where users can view the selected names.
 Finally, you'll change the app's theme color.
 
 
-#### [Building Beautiful UIs with Flutter]({{site.codelabs}}/codelabs/flutter)
+#### [Write your first Flutter app on the web][]
+
+Implement a simple web app in DartPad (no downloads
+required!) that displays a sign-in screen
+containing three text fields. As the user fills out the
+fields, a progress bar animates along the top of the
+sign-in area. This codelab is written specifically for
+the web, but if you have downloaded and configured
+Android and iOS tooling, the completed app just
```

Comment:
The end of the sentence is a little confusing — _just_ could be read as _only_, e.g. How about:

...the completed app works on Android and iOS devices, as well.